src/test/java/gov/usgs/cida/pubs/busservice/CrossRefBusServiceTest.java

```diff
@@ -246,16 +247,35 @@ public void tetBuildCrossRefPost() throws IOException {
 
 	@Test
 	public void getGoodCrossrefXml() throws XMLValidationException, IOException {
-		when(contributorTypeDao.getById(ContributorType.AUTHORS)).thenReturn(buildContributorTypeAuthor());
```

Comment:
Can the PublicationBusService.class & ContributorType.class be removed from the @SpringBootTest and contributorTypeDao also be removed?

Reply:
yup